DirectProgramming/DPC++FPGA/Tutorials/Features/ac_types/ac_int/sample.json

```diff
@@ -0,0 +1,55 @@
+{
+  "guid": "394849F1-EDB6-4234-9276-B5A2A9DD6D5A",
```

Comment:
double-checking this is unique.

Reply:
Yup, created from the GUID generated linked in the code samples docs.